lms/djangoapps/program_enrollments/signals.py

```diff
@@ -44,6 +45,25 @@ def listen_for_social_auth_creation(sender, instance, created, **kwargs):  # lin
         raise
 
 
+def generate_default_display_name(self):
+    """
+    Returns a default display name for SamlProviderConfig.
+    """
+    t = datetime.datetime.now()
+    return f'{self.prefix}-{self.slug}-{t.minute}{t.second}'
```

Comment:
we may want to make the display name field longer - slug used here could be 30 char on its own

Reply:
Good point, I think removing the prefix and making display name 35 (30 + 5 characters from -mmss) is a good solution